functional/FunctionalTest.java

```diff
@@ -3119,45 +3122,22 @@ public static void composeObject_test1() throws Exception {
       List<ComposeSource> listSourceObjects = new ArrayList<ComposeSource>();
       listSourceObjects.add(s1);
       listSourceObjects.add(s2);
-
-      client.composeObject(bucketName, destinationObjectName, listSourceObjects, null, null);
-      Files.delete(Paths.get(filename1));
-      Files.delete(Paths.get(filename2));
-
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename1).build());
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename2).build());
-      client.removeObject(
-          RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
-
-      mintSuccessLog(
-          "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, "
-              + "Map <String,String > headerMap, ServerSideEncryption sseTarget)",
-          "size: 6 MB & 6 MB ",
-          startTime);
-    } catch (Exception e) {
-      ErrorResponse errorResponse = null;
-      if (e instanceof ErrorResponseException) {
-        ErrorResponseException exp = (ErrorResponseException) e;
-        errorResponse = exp.errorResponse();
+      try {
+        client.composeObject(bucketName, destinationObjectName, listSourceObjects, null, null);
+        client.removeObject(
+            RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
+      } finally {
+        Files.delete(Paths.get(filename1));
+        Files.delete(Paths.get(filename2));
+        client.removeObject(
+            RemoveObjectArgs.builder().bucket(bucketName).object(filename1).build());
+        client.removeObject(
+            RemoveObjectArgs.builder().bucket(bucketName).object(filename2).build());
```

Comment:
Since this whole finally block is being repeated multiple times (sometimes with 2 files, sometimes with 1), it can be extracted into a method that takes an array/list of filenames as argument and deletes them from both the local filesystem and the object store. Something like:

`private void deleteFilesAndObjects(String filenames[]);`